app/styles/app.scss

```diff
@@ -323,10 +334,81 @@ input.light {
 
  }
 
+ .employee-button {
+   width: 119px;
+   height: 30px;
+   background-color: #fff;
+   font-weight: 400;
+   font-size: 14px;
+   line-height: 18px;
+   margin:10px;
+   margin-right:30px;
+
+ }
+
+ .add-employee-slide-container {
+   right:0;
+   top:0;
+   position: absolute;
+   width: 100%;
+   height:100%;
+   z-index: 10;
+   
+   opacity:1 ;
+   display: flex;
+   justify-content: space-around;
+   @include animation(slide-modal .1s 1);
+ 
+   .add-employee-outerslide{ 
+      background: #12344D;
+      opacity: 0.3;
+      width: 45%;   
+   }
+
+   .add-employee-slide {
+      width: 55%;
+      background-color: #fff;  
+   }
+ }
+ 
+.slideheader {
+   margin:10px;
+   margin-left: 15px;
+   justify-content: space-between;
+}
+
+.add-employee-slide-btns {
+   display: flex;
 
+   .apply-btn,.reset-btn {
+      padding: 10px;
+      font-weight: 400;
+   }
 
+   .apply-btn:hover, .reset-btn:hover {
+      background:#12344D ;
+      color: #fff;
+   }
 
+}
+
+.slide-headertext {
+   padding: 10px;
+}
 
+@keyframes slide-modal {
```

Comment:
This file is getting big. We can spit it module separated css files and import them. For example, all slider related css can be written in slider.scss.